.github/workflows/scripts/scanner-update-nvd-legacy.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+"""
+Fetch NVD data from the JSON 1.1 feed archives, parse them into API 2.0
+schema and save into files.
+"""
+
+import gzip
+import json
+import logging
+import os
+from urllib import request
+from urllib import parse
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s: %(levelname)s: %(name)s %(message)s")
+
+
+def convert_cve_feed_to_api(cve_feed):
+    """Convert NVDCVEFeedJSON10DefCVEItem to CVEAPIJSON20DefCVEItem."""
+
+    # Required fields.
+
+    cve_api = {"id": cve_feed['cve']['CVE_data_meta']['ID'],
+               "published": cve_feed['publishedDate'],
+               "lastModified": cve_feed['lastModifiedDate'],
+               "descriptions": [],
+               "metrics": {}}
+
+    # Description.
+
+    for desc in cve_feed['cve'].get('description', {}).get('description_data', []):
```

Comment:
nit: can ignore anything not in English

Reply:
actually I think JSON only has English

Reply:
Left like this to minimize noise because the updater filter things out already, can be updated later also.